eventmetrics/parsers/enums/validationType.go

```diff
@@ -0,0 +1,96 @@
+package enums
+
+import "strings"
+
+type ValidationType int
+
+const (
+	UnknownValidation ValidationType = iota
+	BootTimeValidation
+	BirthdateValidation
```

Comment:
I thought it might be confusing to just have BootTime or Birthdate since there's an Event field that is already BootTime and Birthdate, but you're right-should be consistent with all the enums!